pandas/core/algorithms.py

```diff
@@ -903,7 +904,8 @@ def value_counts_internal(
     else:
         if is_extension_array_dtype(values):
             # handle Categorical and sparse,
-            result = Series(values, copy=False)._values.value_counts(dropna=dropna)
+            print(Series(values, copy=False)._values)
```

Comment:
U have a print statement in your code

Reply:
Sorry for that. I will remove it and update pull request.
Thanks @samukweku.